src/wbase/FileChannelShared.h

```diff
@@ -105,14 +104,8 @@ class FileChannelShared {
      */
     static nlohmann::json filesToJson(std::vector<QueryId> const& queryIds, unsigned int maxFiles);
 
-    /// The factory method for the channel class.
-    static Ptr create(std::shared_ptr<wbase::SendChannel> const& sendChannel, qmeta::CzarId czarId,
-                      std::string const& workerId = std::string());
-
     /// The factory method for handling UberJob over http.
-    static Ptr create(std::shared_ptr<wbase::UberJobData> const& uberJob, qmeta::CzarId czarId,
-                      std::string const& czarHostName, int czarPort,
-                      std::string const& workerId);  // TODO:UJ delete all params except uberJob
+    static Ptr create(std::shared_ptr<wbase::UberJobData> const& uberJob);
```

Comment:
The name of the parameter `uberJob` doesn't match the type `UberJobData`. It may need to be replaced with `uberJobData`?